tests/integration-tests/tests/monitoring/test_monitoring.py

```diff
@@ -190,3 +195,15 @@ def _verify_alarms(alarms, metric_name, instance_id):
         assert_that(alarms[0]["Dimensions"]).contains({"Name": "path", "Value": "/"}).contains(
             {"Name": "InstanceId", "Value": instance_id}
         )
+
+
+def _test_detailed_monitoring(ec2_client, compute_instance_ids):
+    ec2_response = ec2_client.describe_instances(InstanceIds=compute_instance_ids)
```

Comment:
the ec2 client is not used outside the function, what about we define it only in the function and pass the region in:
```
ec2_response = boto3.client("ec2", region_name=region).describe_instances(InstanceIds=InstanceIds[compute_instance_ids[0]])
```

Reply:
yeah that will work as well